content/Animations in React Native - Fernanda Toledo.md

```diff
@@ -0,0 +1,116 @@
+
+![](https://media.giphy.com/media/QTxxds3ZqTedzYYFFr/giphy.gif)
+
+https://gist.github.com/fernandatoledo/cf154351050339d862798b043c3cfeb9
+
+https://gist.github.com/fernandatoledo/8b136ef05d67ac73836a8615130f68ca
+
+In this example multiple animations are taking place in parallel, there's a vertical and horizontal translation, along with rotation.
+
+#### Our Library
+
+In order to make our hooks readily available to any developer that wants to start their React Native animations journey, we created a library that contains the hooks described in this article. You can try it out by just running on your react native project folder:
+```
+yarn add @rootstrap/react-native-use-animate
+```
+Or
+```
+npm install @rootstrap/react-native-use-animate --save
+```
+
+#### Alternatives
+
+While reading this post you might have asked yourself, are there any alternative libraries out there that do this? And the answer is yes. To name a few of the most well-known ones we have [react-native-reanimated](https://github.com/software-mansion/react-native-reanimated), [react-native-animatable](https://github.com/oblador/react-native-animatable), [react-native-motion](https://github.com/xotahal/react-native-motion), [Lottie](https://airbnb.io/lottie).
+
+They all serve different purposes and depending on what you need to implement you might decide to go for one or the other. One thing that they all have in common is that they cover a lot of ground with their APIs and that is good. At the same time if you are just getting started and want to do a simple animation there are two drawbacks that we found. First, kind of the same issue we mentioned about React Native's own Animated API, there are too many options and a lot to read in order to get used to it which could be overwhelming. Second, these are rather big libraries, some bigger than others but overall, they all pack things that are not needed if you are just looking to lift up your application a bit by adding a simple animation here and there.
+
+This is why we believe that our library has its own distinct purpose and can coexist with the other alternatives out there. As we mentioned before our library is ideal for people that are just getting started or want to implement simple animations.
+
+
+### Summary
+
+Getting started with animations in React Native can be a bit overwhelming so we created a library to simplify this process and make it more inviting for developers that might have been on the fence about adding animations to their apps.
+
+We hope that the React Native community starts using this library and that we start getting feedback so we can continue to improve the library and provide a better experience for beginners.
+
+Remember that to get started you can just simply add the library by using `npm` or `yarn`.
+
+You can always check the source code here and contributions are always welcomed.
+
+Please leave a comment, we would love to know your thoughts on this.
```

Comment:
Maybe I'd mention one last time where they can get the library, like a link to npm for example, just to make it more visible.

Reply:
We wrote our article on drive and then when we passed it here I forgot to fix the <here> link to go to the repo.